go.mod

```diff
@@ -32,6 +32,8 @@ replace github.com/gorilla/websocket v1.4.0 => github.com/gorilla/websocket v1.4
 
 replace github.com/coreos/etcd v3.3.17+incompatible => github.com/coreos/etcd v3.3.25+incompatible
 
+replace github.com/coreos/etcd v3.3.13+incompatible => github.com/coreos/etcd v3.3.25+incompatible
```

Comment:
Could we have a single replace like:

```
replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.25+incompatible
```

Reply:
No, as that could also replace newer versions with the old version. IMO we should only override exact versions otherwise it becomes hard to reason about what's actually included. There is an upstream issue for `go mod tidy` to remove `replaces` when they are not used